packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -345,19 +375,9 @@ export class NotifiFrontendClient {
         );
         return signature;
       }
+      case 'INJECTIVE':
       case 'OSMOSIS':
       case 'ZKSYNC':
```

Comment:
I think this (removing entire section here) might break the login section since the following 3 chains  is the type of `NotifiConfigWithPublicKeyAndAddress` wich does not have `walletPublicKey` field. 
- `OSMOSIS`
- `ZKSYNC`
- `INJECTIVE`